drivers/hmis_external_apis/extensions/hmis/unit_type_extension.rb

```diff
@@ -24,6 +24,9 @@ module UnitTypeExtension
       def track_availability(project_id:, user_id:)
         return unless HmisEnforcement.hmis_enabled? && HmisExternalApis::AcHmis::Mper.enabled? && mper_id
 
+        # Skip if UUID, which means that this was a project created within the HMIS.
+        return if project_id.size == 32
```

Comment:
This seems slightly odd, but given the scope of what it's doing is probably fine.